x-pack/qa/multi-project/core-rest-tests-with-multiple-projects/build.gradle

```diff
@@ -39,6 +39,7 @@ tasks.named("yamlRestTest").configure {
     '^cat.snapshots/*/*',
     '^cluster.desired_balance/10_basic/*',
     '^data_stream/40_supported_apis/Verify shard stores api', // uses _shard_stores API
+    '^data_stream/230_data_stream_options.yml', // updating data stream options does not yet support multi project
```

Comment:
I will make data stream options support multi-project in a follow up PR to keep this within scope.